pkgs/development/python-modules/http-ece/default.nix

```diff
@@ -18,25 +19,33 @@ buildPythonPackage rec {
     hash = "sha256-tZIPjvuOG1+wJXE+Ozb9pUM2JiAQY0sm3B+Y+F0es94=";
   };
 
-  postPatch = ''
-    substituteInPlace setup.py \
-      --replace-fail '"nose",' "" \
-      --replace-fail '"coverage",' ""
-  '';
+  patches = [
+    (fetchpatch2 {
```

Comment:
Could you link the PR here?

Reply:
It's https://github.com/web-push-libs/encrypted-content-encoding/pull/76, and since there's an active discussion currently with the maintainer about merging it, I'll mark this as draft, and if it gets merged I'll update the patch here(or just fetchfromgithub the commit instead)

Reply:
PR merged, no need to link here anymore.